lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -223,9 +223,22 @@ def add_payment_token(source, payment, options)
         end
       end
 
+      def add_decypted_wallet(source, payment, options)
+        source[:sourceType] = 'DecryptedWallet'
+        source[:card] = {
+          cardData: payment.number,
+          expirationMonth: (format(payment.month, :two_digits) if payment.month),
+          expirationYear: (format(payment.year, :four_digits) if payment.year)
+        }
```

Comment:
In my last PR I received an observation to change a similar hash to:
```
card = source[:card] = {}
card[:cardData] = payment.number
card[:expirationMonth] = format(payment.month, :two_digits) if payment.month
card[:expirationYear] = format(payment.year, :four_digits) if payment.year
```
I recommend you make this change

Related comment: https://github.com/activemerchant/active_merchant/pull/4632#discussion_r1033952778

Reply:
Ok, then I will use the same pattern